<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QR Code Scanner</string>
  </property>
  <widget class="QWidget" name="centralwidget">

   <!-- Layout for text details -->
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>130</y>
      <width>431</width>
      <height>351</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTextEdit" name="textEdit">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string>border: 2px solid #D3D3D3; border-radius: 5px; padding: 10px;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>

   <!-- Layout for QR code image -->
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>451</width>
      <height>351</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string/>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>300</height>
        </size>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>

   <!-- Partition Line between QR code and Text Details -->
   <widget class="QFrame" name="line">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>120</y>
      <width>20</width>
      <height>380</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::VLine</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
   </widget>

   <!-- Decode Button -->
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>520</y>
      <width>311</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
      background-color: #5C9DED; 
      color: white; 
      border-radius: 10px; 
      padding: 15px; 
      border: 2px solid #3D8BFF;
     </string>
    </property>
    <property name="text">
     <string>Decode QR Code</string>
    </property>
   </widget>

   <!-- ComboBox for Language Selection -->
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>60</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string notr="true">English</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hindi</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gujarati</string>
     </property>
    </item>
   </widget>

   <!-- Label for Language Selection -->
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Select Language :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter</set>
    </property>
   </widget>

  </widget>

  <!-- Menu Bar and Status Bar -->
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>